9-spanner/guestbook-service/src/main/java/com/example/guestbook/GuestbookMessageRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.example.guestbook;
+
+import java.util.List;
+
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+@RepositoryRestResource
+public interface GuestbookMessageRepository extends
+	PagingAndSortingRepository<GuestbookMessage, Long> {
```

Comment:
hiya! quick question, does this work w/ the UUID string? I think the ID type needs to be updated here.